motionplan/tpSpaceRRT_test.go

```diff
@@ -355,7 +355,8 @@ func TestPtgCheckPlan(t *testing.T) {
 	})
 	t.Run("verify partial plan with non-nil errorState and obstacle", func(t *testing.T) {
 		// create obstacle which is behind where the robot already is, but is on the path it has already traveled
-		box, err := spatialmath.NewBox(spatialmath.NewZeroPose(), r3.Vector{10, 10, 1}, "obstacle")
+		box, err := spatialmath.NewBox(
+			spatialmath.NewPoseFromPoint(r3.Vector{-100, 0, 0}), r3.Vector{10, 10, 1}, "obstacle")
```

Comment:
If the position of the box is (0,0,0) and we are interpreting this to be relative to the base, then this means the box is currently in collision with the base since they are right on top of each other. 

Previously, the box being at (0,0,0) meant that it was behind the base since it was at some (x,y,0) since we are checking the remainder of a plan.